SessionUIKit/Components/SwiftUI/Toast.swift

```diff
@@ -34,7 +34,7 @@ public struct ToastModifier: ViewModifier {
         }
   
         workItem = task
-        DispatchQueue.main.asyncAfter(deadline: .now() + 5, execute: task)
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1.5, execute: task)
```

Comment:
This doesn't need to be addressed as part of this PR (since it also affects non-SwiftUI toasts) but I was just wondering whether the duration should be calculated based on the length of the toast message? (for something like "Copied" `1.5` seconds is fine but for something like "Failed to send invitations to Bob and Alice in group TestGroup" 1.5 seconds might be too short 🤔)

Reply:
Yeah, this change is just making the duration of toast the same as non-SwiftIU toasts if I remember it correctly. 5 seconds is too long for some case. But I totally agree with you, we should make the duration depending on the length of the message that we want to show in the toast. I'll double check the logic on non-SwiftIU toasts.